<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Messdaten</class>
 <widget class="QDialog" name="Messdaten">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1181</width>
    <height>671</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1181</width>
     <height>671</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="Messung">
    <attribute name="title">
     <string>Messung</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>140</y>
       <width>281</width>
       <height>433</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Port</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Analog 0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Analog 1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Analog 2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Analog 3</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Analog 4</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Analog 5</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Analog 6</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Analog 7</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Digital 0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Digital 1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Digital 2</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Digital 3</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Digital 4</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>Digital 5</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>Digital 6</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>Digital 7</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_18">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Wert</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="analog_0"/>
        </item>
        <item>
         <widget class="QLineEdit" name="analog_1"/>
        </item>
        <item>
         <widget class="QLineEdit" name="analog_2"/>
        </item>
        <item>
         <widget class="QLineEdit" name="analog_3"/>
        </item>
        <item>
         <widget class="QLineEdit" name="analog_4"/>
        </item>
        <item>
         <widget class="QLineEdit" name="analog_5"/>
        </item>
        <item>
         <widget class="QLineEdit" name="analog_6"/>
        </item>
        <item>
         <widget class="QLineEdit" name="analog_7"/>
        </item>
        <item>
         <widget class="QLineEdit" name="digital_0"/>
        </item>
        <item>
         <widget class="QLineEdit" name="digital_1"/>
        </item>
        <item>
         <widget class="QLineEdit" name="digital_2"/>
        </item>
        <item>
         <widget class="QLineEdit" name="digital_3"/>
        </item>
        <item>
         <widget class="QLineEdit" name="digital_4"/>
        </item>
        <item>
         <widget class="QLineEdit" name="digital_5"/>
        </item>
        <item>
         <widget class="QLineEdit" name="digital_6"/>
        </item>
        <item>
         <widget class="QLineEdit" name="digital_7"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>281</width>
       <height>52</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_19">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Datum</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="line_date"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_20">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Uhrzeit</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="line_time"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>300</y>
       <width>421</width>
       <height>25</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QComboBox" name="combo_values"/>
      </item>
      <item>
       <widget class="QPushButton" name="button_delete">
        <property name="text">
         <string>Löschen</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QCalendarWidget" name="calendarWidget">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>100</y>
       <width>248</width>
       <height>183</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>190</y>
       <width>201</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Nur ausgewähltes Datum anzeigen</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="CAN">
    <attribute name="title">
     <string>CAN</string>
    </attribute>
    <widget class="QTableWidget" name="can_table">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>561</width>
       <height>561</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>50</y>
       <width>561</width>
       <height>331</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QCalendarWidget" name="calender_start"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_22">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Bis :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTimeEdit" name="time_end"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_21">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Von : </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTimeEdit" name="time_start"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QCheckBox" name="check_can">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>390</y>
       <width>141</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Alle Nachrichten anzeigen</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_delete_can">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>410</y>
       <width>161</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Löschen</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
